cv_next/app/cv/[cvId]/components/commentSection/comment.tsx

```diff
@@ -24,14 +23,29 @@ interface NewCommentBlockProps {
   setCommentOnComment: (value: string) => void;
   addNewCommentClickEvent: () => Promise<void>;
   setCommentOnCommentStatus: (status: boolean) => void;
+  parentCommenter: string;
 }
 
 const NewCommentBlock = ({
   commentOnCommentStatus,
   setCommentOnComment,
   addNewCommentClickEvent,
   setCommentOnCommentStatus,
+  parentCommenter,
 }: NewCommentBlockProps) => {
+  const inputRef = useRef<HTMLInputElement>(null);
+  const [inputValue, setInputValue] = useState(`@${parentCommenter} `);
+
+  useEffect(() => {
+    if (commentOnCommentStatus && inputRef.current) {
+      inputRef.current.focus();
+    }
+  }, [commentOnCommentStatus]);
```

Comment:
I'm not getting any warning here and when trying to add it anyway:
```
React Hook useEffect has an unnecessary dependency: 'inputRef.current'. Either exclude it or remove the dependency array. Mutable values like 'inputRef.current' aren't valid dependencies because mutating them doesn't re-render the component.
```